### ISSUE TITLE
Prompt before navigate

### ISSUE THREAD

**Issue by garronej** [zilch/type-route#104]
Hi!  

It would be nice to have a way to prompt a message before navigating to a new page when the use haven't submitted the form yet.  

https://stackoverflow.com/a/49163569/3731798

Best

**Comment by bradenhs** [comment]
When you create a router it returns a session object with a method called `unblock`. Would this satisfy your use case?
https://zilch.dev/type-route/docs/guides/preventing-navigation

**Comment by garronej** [comment]
Perfect thank you!
Sorry I didn't find it myself.
